communication/kubernetesio-x-guidelines.md

```diff
@@ -0,0 +1,43 @@
+# @kubernetesio X Account guidelines
+
+SIG ContribEx Comms has multiple communication channels, not limited to the kubernetesio account. The main difference is the targeted audience.
+
+### Kubernetes Contributors can be targeted by the following channels:
+
+* [dev@kubernetes.io](mailto:dev@kubernetes.io) mailing list
+* Kubernetes Slack Announcement Bot
+* Mastadon: [k8sContributors@hachiderm](https://hachyderm.io/@K8sContributors)
+* X: [k8sContributors](https://x.com/k8sContributors)
+
+### End User focused Channels are
+
+* X: [kubernetesio](https://x.com/kubernetesio)
+* [LinkedIn](https://www.linkedin.com/company/kubernetes/posts/)
+
+### Mixed Channels:
+
+* [Lwkd Newsletter](https://lwkd.info/)
+
+## The following policy will describe the usage policy of the kubernetesio X account.
+
+* The kubernetesio account is reaching hundreds of thousands people. As such, we need to make sure we link to things that are owned by the community itself (mailing lists, GitHub, surveys, etc.).
+    * As a rule, we should not link to individuals' Twitter accounts or other personal properties given the massive amount of attention it could create
```

Comment:
```suggestion
    * As a general rule, we should avoid linking to individual Twitter accounts or other personal properties given the massive amount of attention it may generate.
```